### ISSUE TITLE
Memory leak?

### ISSUE THREAD

**Issue by MathBunny** [adam-p/markdown-here#339]
I've just experienced a memory leak where MarkDown-here used up 3GB of RAM. Did anyone else experience this?

**Comment by leesei** [comment]
I'm on Chrome 49.0.2623.108 (64-bit) and have the same issue.

**Comment by jakekei** [comment]
Same here with Chrome version 49.0.2623.87 (on win 7 platform, 64 bits)

**Comment by cben** [comment]
Same here, Chrome 49.0.2623.112 (ubuntu 64-bit):
![markdown-here-leak](https://cloud.githubusercontent.com/assets/273688/15276427/71ccbabe-1af0-11e6-9a34-b0a1cefb5320.png)
That 2.3G accumulated over today, most of the time Chrome merely run in background (tray icon, no window).

Heap snapshot & allocations (around 1min) profiles:
https://transfer.sh/JK8jl/heap-20160515t230105.heapsnapshot
https://transfer.sh/14eXYU/heap-20160515t230117.heaptimeline
Didn't see a clear culprit in the profiles though.

**Comment by jbulava** [comment]
Yes, 3GB of RAM and full fan speed. Didn't experience it until I upgraded to El Capitan, but that's probably a coincidence given the comments above about Windows and Ubuntu.  I toggle the extension on and off only when needed.

MacBook Pro (Retina, 13-inch, Early 2015)
OS X El Capitan 10.11.5
Processor: 3.1 GHz Intel Core i7
Chrome: Version 50.0.2661.102 (64-bit)

**Comment by emmaly** [comment]
I know this is old, but it feels like the most likely place to put this.  The extension was using more than 9GB of RAM on my Pixelbook.  I've disabled it many times in the past, but I really need to convert Markdown pretty often.  Screenshot below.

Extension version 2.12.0
Chrome OS version 77.0.3865.56 (Official Build) beta (64-bit)
Google Pixelbook, i5, 8GB RAM

![screenshot](https://user-images.githubusercontent.com/89385/64100062-a48e9a00-cd1f-11e9-8b0b-edcdcb821e7d.png)

**Comment by daniel-stoneuk** [comment]
![image](https://user-images.githubusercontent.com/15860949/65342088-e6e22300-dbc9-11e9-893d-e128dce6107e.png)

Same here with my Pixel Slate. Been wondering why it's been charging slow and running so warm and I just found it using so much ram and CPU. (sorry for the non screenshot, killed it immediately after taking a snap).

Don't have this issue on other platforms, could it be an isolated ChromeOS issue?

Version 77.0.3865.63 (Official Build) beta (64-bit)
